modules/platform/dev.galasa.platform/build.gradle

```diff
@@ -36,9 +36,9 @@ dependencies {
 
         api 'com.auth0:java-jwt:4.4.0' // used by wrapper.
         
-        api 'com.fasterxml.jackson.core:jackson-annotations:2.16.0' // used by wrapper.
-        api 'com.fasterxml.jackson.core:jackson-core:2.16.0' // used by wrapper.
-        api 'com.fasterxml.jackson.core:jackson-databind:2.16.0' // used by wrapper.
+        api 'com.fasterxml.jackson.core:jackson-annotations:2.20' // used by wrapper.
+        api 'com.fasterxml.jackson.core:jackson-core:2.20.0' // used by wrapper.
+        api 'com.fasterxml.jackson.core:jackson-databind:2.20.0' // used by wrapper.
 
```

Comment:
Note - MEND scan for v0.46.0 might be ugly now we've changed the platform, will have to think of a workaround